qbraid/runtime/schemas/job.py

```diff
@@ -166,32 +169,31 @@ def from_dict(cls, job_data: dict[str, Any]) -> RuntimeJobModel:
         Returns:
             RuntimeJobModel: An instance of RuntimeJobModel.
         """
-        experiment_type = ExperimentType(job_data.pop("experimentType", "gate_model"))
-        job_data["experimentType"] = experiment_type
-
+        # Set experiment type
+        job_data["experimentType"] = ExperimentType(
+            "annealing"
+            if "anneal" in job_data.get("qbraidDeviceId")
+            else job_data.pop("experimentType", "gate_model")
+        ).value
```

Comment:
This convention is going break the minute we add another device e.g. D-Wave. Should probably add experiment type to API job schema